sklearn/datasets/_kddcup99.py

```diff
@@ -174,6 +184,18 @@ def fetch_kddcup99(*, subset=None, data_home=None, shuffle=False,
     if return_X_y:
         return data, target
 
+    if as_frame:
+        frame, X, y = _convert_data_dataframe("fetch_kddcup99",
+                                              data,
+                                              target,
+                                              feature_names,
+                                              target_names)
+        return Bunch(data=X,
```

Comment:
you always return this bunch. We only need to add `frame=None` before the `as_frame` if statement.
We could update the documentation as well that we will always return `feature_names` and `target_names` in the `Bunch`

Reply:
DONE.